resolwe/flow/execution_engines/workflow/__init__.py

```diff
@@ -0,0 +1,106 @@
+"""An execution engine that supports workflow specifications."""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import collections
+
+from six import string_types
+
+from django.db import transaction
+
+from resolwe.flow.execution_engines.base import BaseExecutionEngine
+from resolwe.flow.execution_engines.exceptions import ExecutionError
+from resolwe.flow.expression_engines import EvaluationError
+from resolwe.flow.models import Data, Process
+
+
+class ExecutionEngine(BaseExecutionEngine):
+    """An execution engine that supports workflow specifications."""
+
+    name = 'workflow'
+
+    def get_output_schema(self, process):
+        """Return any additional output schema for the process."""
+        return [
+            {'name': 'steps', 'type': 'list:data:'},
```

Comment:
No. Because these are the data objects that the steps can output and they can be of arbitrary types. I needed to change the type schema in order to allow definition of a generic data type that can match any data object.